pkg/virtctl/guestfs/guestfs_test.go

```diff
@@ -140,14 +152,40 @@ var _ = Describe("Guestfs shell", func() {
 			cmd := testing.NewRepeatableVirtctlCommand(commandName, pvcName, "--root=true", "--uid=1001")
 			err := cmd()
 			Expect(err).To(HaveOccurred())
-			Expect(err.Error()).Should(Equal(fmt.Sprintf("cannot set uid if root is true")))
+			Expect(err.Error()).Should(Equal("cannot set uid if root is true"))
```

Comment:
Since you've started touching this. A more appropriate matcher for this is `MatchError()`. You can replace both lines L154 and L155 with it.